src/shadow-embed.js

```diff
@@ -75,7 +54,10 @@ export function createShadowRoot(hostElement) {
   }
 
   // Native support.
-  if (isShadowDomSupported()) {
+  const shadowDomSupported = getShadowDomSupportedVersion();
+  if (shadowDomSupported == ShadowDomVersion.V1) {
+    return hostElement.attachShadow({mode: 'open'});
```

Comment:
So, yeah, `attachShadow` looks to be absent is externs :(

Please try to add this to [shadow_dom.js](https://github.com/ampproject/amphtml/blob/master/third_party/closure-compiler/externs/shadow_dom.js):

```
/**
 * @see https://www.w3.org/TR/shadow-dom/#idl-def-ShadowRootInit
 * @typedef {{
 *   mode: string
 * }}
 */
var ShadowRootInit;

/**
 * @see https://www.w3.org/TR/shadow-dom/#h-extensions-to-element-interface
 * @param {ShadowRootInit=} opt_init
 * @return {!ShadowRoot}
 */
Element.prototype.attachShadow = function(opt_init) {};
```

This should do it.